regex-assembly/942152.ra

```diff
@@ -0,0 +1,8 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##!+ i
```

Comment:
Same as above: prefix and suffix need to be part of the include file.

Reply:
Moved them. Thank you for fixing the crs-toolchain!